typescript/s3-object-lambda/lib/s3-object-lambda-stack.ts

```diff
@@ -24,85 +27,87 @@ export class S3ObjectLambdaStack extends cdk.Stack {
-        code: lambda.Code.fromAsset('resources/retrieve-transformed-object-lambda')
-      }
+      runtime: lambda.Runtime.NODEJS_14_X,
+      handler: 'index.handler',
+      code: lambda.Code.fromAsset('resources/retrieve-transformed-object-lambda')
+    }
     );
 
     // Object lambda s3 access
     retrieveTransformedObjectLambda.addToRolePolicy(new iam.PolicyStatement({
-        effect: iam.Effect.ALLOW,
-        resources: ['*'],
-        actions: ['s3-object-lambda:WriteGetObjectResponse']
-      }
+      effect: iam.Effect.ALLOW,
+      resources: ['*'],
+      actions: ['s3-object-lambda:WriteGetObjectResponse']
+    }
     ));
     // Restrict Lambda to be invoked from own account
     retrieveTransformedObjectLambda.addPermission('invocationRestriction', {
       action: 'lambda:InvokeFunction',
       principal: new iam.AccountRootPrincipal(),
-      sourceAccount: cdk.Aws.ACCOUNT_ID
+      sourceAccount: Aws.ACCOUNT_ID
     });
 
     // Associate Bucket's access point with lambda get access
-    const policyDoc = new iam.PolicyDocument();
-    const policyStatement = new iam.PolicyStatement({
-      effect: iam.Effect.ALLOW,
-      actions: ['s3:GetObject'],
-      principals: [
-        new iam.ArnPrincipal(<string>retrieveTransformedObjectLambda.role?.roleArn)
-      ],
-      resources: [`${accessPoint}/object/*`]
+    const policyDoc = new iam.PolicyDocument({
+      statements: [
+        new iam.PolicyStatement({
+          sid: 'AllowLambdaToUseAccessPoint',
+          effect: iam.Effect.ALLOW,
+          actions: ['s3:GetObject'],
+          principals: [
+            new iam.ArnPrincipal(<string>retrieveTransformedObjectLambda.role?.roleArn)
+          ],
+          resources: [`${accessPoint}/object/*`]
+        })
+      ]
     });
-    policyStatement.sid = 'AllowLambdaToUseAccessPoint';
-    policyDoc.addStatements(policyStatement);
 
     new s3.CfnAccessPoint(this, 'exampleBucketAP', {
-        bucket: bucket.bucketName,
-        name: S3_ACCESS_POINT_NAME,
-        policy: policyDoc
-      }
+      bucket: bucket.bucketName,
+      name: S3_ACCESS_POINT_NAME,
+      policy: policyDoc
+    }
     );
 
     // Access point to receive GET request and use lambda to process objects
     const objectLambdaAP = new s3ObjectLambda.CfnAccessPoint(this, 's3ObjectLambdaAP', {
-        name: OBJECT_LAMBDA_ACCESS_POINT_NAME,
-        objectLambdaConfiguration: {
-          supportingAccessPoint: accessPoint,
-          transformationConfigurations: [{
-            actions: ['GetObject'],
-            contentTransformation: {
-              'AwsLambda': {
-                'FunctionArn': `${retrieveTransformedObjectLambda.functionArn}`
-              }
+      name: OBJECT_LAMBDA_ACCESS_POINT_NAME,
+      objectLambdaConfiguration: {
+        supportingAccessPoint: accessPoint,
+        transformationConfigurations: [{
+          actions: ['GetObject'],
+          contentTransformation: {
+            'awsLambda': {
```

Comment:
Can we revert the casing please? This was a breaking change that will get patched on the next release

Reply:
Sure, it's now back to previous casing